trail/upload/tests/test_processors.py

```diff
@@ -10,11 +10,25 @@
 from upload.process_uploads.upload_wrapper import TemporaryUploadedFileWrapper
 from upload.process_uploads.upload_processor import UploadProcessor
 from upload.process_uploads.fits_processor import FitsProcessor
+import upload.process_uploads.header_standardizer as header_standardizer
 
 
 TESTDIR = os.path.abspath(os.path.dirname(__file__))
 
 
+class MockAstrometryServer:
+    """Mock Astrometry function that acts like the astrometry.net server"""
+    def __init__(self, return_no_solution=False):
+        self.return_no_solution = return_no_solution
+
+    def solve_from_image(self, path_to_file, preprocess=True, solve_timeout=120):
```

Comment:
Might consider adding a comment that `preprocess` is here to just match the call signature in `header_standardizer.py`, optionally to make this future-proof remove `preprocess` and add `**kwargs`. In that way it will never matter what the actual full signature of the mocked function could ever be - just what are key arguments you need in your tests.